sdks/python/apache_beam/options/value_provider_test.py

```diff
@@ -34,6 +34,14 @@
 # <file name acronym>_non_vp_arg<number> for non-value-provider arguments.
 # The number will grow per file as tests are added.
 class ValueProviderTests(unittest.TestCase):
+  def setUp(self):
+    # Reset runtime options.
+    RuntimeValueProvider.set_runtime_options(None)
```

Comment:
Probably document why we need this in both if we actually do (to make sure that someone doesn't drop one).

Reply:
fixed